src/indexv3.js

```diff
@@ -0,0 +1,58 @@
+var interviewApp = {};
+(function(){
+    const container = document.getElementById(`app`);
+    let module = null;
+    
+    
+    const checkIfLogged = function(){
+        const isLogged = function() {
+            return !!window.localStorage.getItem('userData');
+        }
+        //console.log(isLogged())
+        if(!isLogged()){
+            return 'login'
+        }
+        return ''
+    }
+
+
+    interviewApp.navigate = function(page){
+        //console.log(page)
+        if (typeof page !== 'string') {
+            page = 'login';
+        }
+        if(module){
+            module.destroy();
+        }
+        if(checkIfLogged() != ''){
+            page = checkIfLogged();
+        }
+        module = interviewApp[page];
+
+
+        
+            if(localStorage.getItem("evaluation") !== null){
+                q = JSON.parse(localStorage.getItem("evaluation")).map(function(element){
```

Comment:
Please note that this will force browsers to a. retrieve information from local storage (so likely read from disk) and b. parse said information twice, making this processing needlessly complicated, since you've only just obtained the information you're after moments before.